XenAdmin/Wizards/NewSRWizard_Pages/SrWizardHelpers.cs

```diff
@@ -40,9 +40,15 @@ namespace XenAdmin.Wizards.NewSRWizard_Pages
 {
     public class SrWizardHelpers
     {
+        public static bool ValidateServerAddress(String address)
+        {
+            Regex re = new Regex(@"^[^/:*?<>|\\]+$");

```

Comment:
I think we have a better way to validate it, see: 
https://github.com/xenserver/xenadmin/blob/master/XenAdmin/Dialogs/OptionsPages/ConnectionOptionsPage.cs#L191

Uri.CheckHostName(string)